<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExerciseScene</class>
 <widget class="QMainWindow" name="ExerciseScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>复习您的假名！</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/Exercise.png</normaloff>:/res/Exercise.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>681</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="text_kanasOfThisGroup">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>31</y>
      <width>105</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>本组假名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_YourAnswer">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>109</y>
      <width>105</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>输入回答：</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>12</y>
      <width>551</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="outPut_Input_1" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="output">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>あ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="input">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="outPut_Input_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="output_2">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>あ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="input_2">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="outPut_Input_3" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="output_3">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>あ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="input_3">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="outPut_Input_4" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="output_4">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>あ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="input_4">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="outPut_Input_5" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="output_5">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>あ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="input_5">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>651</width>
      <height>184</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="exeTimeRemained">
          <property name="font">
           <font>
            <family>方正粗黑宋简体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>剩余练习轮数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="kanaGroupRemained">
          <property name="font">
           <font>
            <family>方正粗黑宋简体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>本轮练习剩余组数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="allRightAnswer">
          <property name="font">
           <font>
            <family>方正粗黑宋简体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>总计正确个数:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="allFalseAnswer">
          <property name="font">
           <font>
            <family>方正粗黑宋简体</family>
            <pointsize>12</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>总计错误个数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="allKanas">
          <property name="font">
           <font>
            <family>方正粗黑宋简体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>全部假名个数：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="thisGroupCorrectNum">
          <property name="font">
           <font>
            <family>方正粗黑宋简体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>本组正确个数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="thisGroupIncorrectNum">
          <property name="font">
           <font>
            <family>方正粗黑宋简体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>本组错误假名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="thisGroupInCorrectNumRightAns">
          <property name="font">
           <font>
            <family>方正粗黑宋简体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>错误假名改正：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="thisGroupFalseAnsOutPut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="thisTimeCorrectPercent">
          <property name="font">
           <font>
            <family>方正粗黑宋简体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>本轮练习正确率：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="thisTimeIncorrectPercent">
          <property name="font">
           <font>
            <family>方正粗黑宋简体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>本轮练习错误率：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ThisTimeAllWrongAns">
          <property name="font">
           <font>
            <family>方正粗黑宋简体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>本轮练习错误：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="thisTimeFalseAnsOutPut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="showBg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>420</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="RemainTimeWIdget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="remainTime">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>showBg</zorder>
   <zorder>line_3</zorder>
   <zorder>text_kanasOfThisGroup</zorder>
   <zorder>text_YourAnswer</zorder>
   <zorder>widget</zorder>
   <zorder>widget_5</zorder>
   <zorder>RemainTimeWIdget</zorder>
   <zorder>remainTime</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionSetting"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>res/Exit.png</normaloff>res/Exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/helpIcon.png</normaloff>:/res/helpIcon.png</iconset>
   </property>
   <property name="text">
    <string> 查看帮助（练习不会暂停）</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/settingIcon.png</normaloff>:/res/settingIcon.png</iconset>
   </property>
   <property name="text">
    <string>设置背景音乐和背景(练习不会暂停)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
